common/annotationsDerived/fr.xml

```diff
@@ -2687,42 +2687,42 @@ Derived short names and annotations, using GenerateDerivedAnnotations.java. See
-		<annotation cp="🚶🏽‍➡" type="tts">personne qui marche vers la droite</annotation>
-		<annotation cp="🚶🏾‍➡">balade | en route | flâner | homme | marche | marcher | nonchalant | personne | personne qui marche | piéton | promenade | se balader</annotation>
-		<annotation cp="🚶🏾‍➡" type="tts">personne qui marche vers la droite</annotation>
-		<annotation cp="🚶🏿‍➡">balade | en route | flâner | homme | marche | marcher | nonchalant | personne | personne qui marche | piéton | promenade | se balader</annotation>
-		<annotation cp="🚶🏿‍➡" type="tts">personne qui marche vers la droite</annotation>
-		<annotation cp="🚶‍♀‍➡">femme | femme qui marche | marche | marcher | piétonne | randonneuse</annotation>
-		<annotation cp="🚶‍♀‍➡" type="tts">femme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏻‍♀‍➡">femme | femme qui marche | marche | marcher | piétonne | randonneuse</annotation>
-		<annotation cp="🚶🏻‍♀‍➡" type="tts">femme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏼‍♀‍➡">femme | femme qui marche | marche | marcher | piétonne | randonneuse</annotation>
-		<annotation cp="🚶🏼‍♀‍➡" type="tts">femme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏽‍♀‍➡">femme | femme qui marche | marche | marcher | piétonne | randonneuse</annotation>
-		<annotation cp="🚶🏽‍♀‍➡" type="tts">femme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏾‍♀‍➡">femme | femme qui marche | marche | marcher | piétonne | randonneuse</annotation>
-		<annotation cp="🚶🏾‍♀‍➡" type="tts">femme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏿‍♀‍➡">femme | femme qui marche | marche | marcher | piétonne | randonneuse</annotation>
-		<annotation cp="🚶🏿‍♀‍➡" type="tts">femme qui marche vers la droite</annotation>
-		<annotation cp="🚶‍♂‍➡">homme | homme qui marche | marche | piéton</annotation>
-		<annotation cp="🚶‍♂‍➡" type="tts">homme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏻‍♂‍➡">homme | homme qui marche | marche | piéton</annotation>
-		<annotation cp="🚶🏻‍♂‍➡" type="tts">homme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏼‍♂‍➡">homme | homme qui marche | marche | piéton</annotation>
-		<annotation cp="🚶🏼‍♂‍➡" type="tts">homme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏽‍♂‍➡">homme | homme qui marche | marche | piéton</annotation>
-		<annotation cp="🚶🏽‍♂‍➡" type="tts">homme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏾‍♂‍➡">homme | homme qui marche | marche | piéton</annotation>
-		<annotation cp="🚶🏾‍♂‍➡" type="tts">homme qui marche vers la droite</annotation>
-		<annotation cp="🚶🏿‍♂‍➡">homme | homme qui marche | marche | piéton</annotation>
-		<annotation cp="🚶🏿‍♂‍➡" type="tts">homme qui marche vers la droite</annotation>
+		<annotation cp="🚶‍➡">balade | droite | en route | flâner | homme | la | marche | marcher | nonchalant | personne | personne qui marche | piéton | promenade | se balader | vers</annotation>
```

Comment:
Seems a little odd to have an article like "la" be a keyword by itself; perhaps we can refine this later.

Reply:
I agree. I was thinking about this, and we could really use a list of per-language 'stop words', like 'la' and 'en' above, that are ignored in matching and can thus be ignored in derivations and vetting. If you think that's a good idea I can file a ticket.

Reply:
I think that is a good idea (I was speculating about something like that), please do file a ticket. Such a list would probably also be useful elsewhere in CLDR/ICU.

Reply:
Filed a ticket and gave it to you for now.